superai/apis/project.py

```diff
@@ -0,0 +1,398 @@
+from __future__ import absolute_import
+
+import logging
+import re  # noqa: F401
+from abc import ABC, abstractmethod
+
+# python 2 and python 3 compatibility library
```

Comment:
Still includes Python2 code, but is not a blocker. Btw. where is `six` dependency declared? If its coming from a third party dependency, maybe we should specify it until removed.

Reply:
I added the dependency